pkg/registry/core/namespace/storage/storage.go

```diff
@@ -88,6 +88,10 @@ func NewREST(optsGetter generic.RESTOptionsGetter) (*REST, *StatusREST, *Finaliz
 	return &REST{store: store, status: &statusStore}, &StatusREST{store: &statusStore}, &FinalizeREST{store: &finalizeStore}, nil
 }
 
+func (r *REST) Store() *genericregistry.Store {
```

Comment:
why did this change?

Reply:
I need to access `r.store` to be able to call `store.DestroyFunc()`. Unfortunately, `store` field is private so I can not access it outside of this package.

Reply:
> I need to access `r.store` to be able to call `store.DestroyFunc()`. Unfortunately, `store` field is private so I can not access it outside of this package.

How about making this a Destroyable instead?  That way you contain your interface.

Reply:
> How about making this a Destroyable instead? That way you contain your interface.

This is still outstanding.

Reply:
Done